CHANGELOG.md

```diff
@@ -62,6 +62,9 @@ Future releases will be documented in this section.
 
 ## Android
 
+### [2.0.26] - 2020-10-19
+- Overall stability improvements and bug fixes.
+
```

Comment:
It would be great to also mention the merchant init data API for the hybrid integration.